### ISSUE TITLE
Add meshes metadata on export as GLB

### ISSUE THREAD

**Issue by Bulisor** [BabylonJS/Exporters#453]
Here is the discussion:
https://forum.babylonjs.com/t/add-metadata-on-glb-export/1046/7.
The ideea is if we export a mesh which has metadata on it, on export as GLB try to keep these metadata somewhere so on import it again to get same infos.

**Comment by Drigax** [comment]
I see. I agree that we should preserve this metadata. Can you make a 3dsMax scene that has metadata that our exporter strips out? 

I'm putting this on my personal backlog to tackle in about 2-3 months, but if anyone wants to fix this before I get to it, feel free :).

**Comment by Bulisor** [comment]
Ok, I'll check this with my 3d artist. 
But this was related to export as glb from code, not from 3dsMax I think.

**Comment by sebavan** [comment]
@Bulisor could you confirm that it is by code cause else I would probably move the issue on the main repo. I would like to not lose it :-)

**Comment by Bulisor** [comment]
Yes. The scenario was: import in the scene an object, .obj, .stl, .babylon, add some infos to its metadata property, export as glb/gltf and use exported file in a different scene. In this new scene to have acces to metadata added before export. Hope it is more clear.

I tried from 3dsMax to export a box with tag, and it is working well.
